w3af/plugins/audit/global_redirect.py

```diff
@@ -54,6 +55,17 @@ def audit(self, freq, orig_response):
 
         :param freq: A FuzzableRequest object
         """
+
+        netloc = freq._uri.netloc
+        netloc = netloc.split(':')[0]
+
+        self.TEST_URLS.extend([netloc + '.' + self.TEST_DOMAIN,
+                               '//' + netloc + '.' + self.TEST_DOMAIN,
+                               'http://' + netloc + '.' + self.TEST_DOMAIN + '/',
```

Comment:
If we add this one, shouldn't we also add one with `https://`? Or maybe chose dynamically based on the protocol used by the fuzzable request?

Reply:
this i did not use dynamically way, because some site only sso is `https`, so it will redirect from `http` to `https` or the other way round, just like `https://sso.xxx.com/login?service=http%3A%2F%2Fwww.test.com%2F`

Reply:
Then, shouldn't we have both, http and https?

Not thinking only about the SSO scenario, but all web apps.